api/Trace/SpanKind.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+interface SpanKind
+{
+    const KIND_INTERNAL = 0;
```

Comment:
Where did the values come from? It couldn't find anything in the specs.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#spankind